### DISCUSSION TITLE
Setup Isso-comments for Django

### DISCUSSION THREAD

**Discussion by ogurec-ogurec** [posativ/isso#1008] [Q&A] [answered]
Hi all. I'm a beginner developer, I want to install _Isso-comments_ on my Django blog. (I use the Windows operating system) 

Please help me with the installation. On the Internet there are only guides for installing Linux and a blog on Flask. It would be great if someone would write here how to install this thing in Django. Help, colleagues)

**Comment by ix5** [comment]
Related: https://github.com/isso-comments/isso/discussions/948 and https://github.com/isso-comments/isso/issues/368

**Reply by ogurec-ogurec** [reply]
@ix5 hi, what am I doing wrong? 

1. **pip install isso** 
2. add '_isso'_ app ot setting _INSTALLED_APPS_. 
3. Create _**isso.conf**_ with the following value: 

```
[general]
dbpath = ./comments.db 
host = 
        127.0.0.1:8000
   	http://127.0.0.1:8000/
	https://localhost:8000
	http://www.localhost:8000
	https://www.localhost:8000


[server]
listen = http://localhost:1234
[admin]
enabled = true
password = password
```
4. where is the embed-min.js file located? I am download to: 
https://github.com/redradishtech/isso/blob/master/isso/js/embed.min.js

5. Added embed.min.js to the static file
6. Added html script to template: 

```
 <script type="module" data-isso="http://127.0.0.1:8000/isso/" src="/static/isso/embed.min.js"></script>
 <section id="isso-thread"></section>
```
7. I get an error in the browser console:

```
       POST http://127.0.0.1:8000/isso/count 404 (Not Found)
       GET http://127.0.0.1:8000/isso/?uri=%2Fquestion%2Fnewpost&nested_limit=5 404 (Not Found)
       POST http://127.0.0.1:8000/isso/preview 404 (Not Found)

```

can I somehow run isso-comments on a local machine (built-in Django server - routing by url) without an HTTP server (nginx)? Or do I really need an HTTP server?

**Comment by ix5** [comment, marked as answer]
Duplicate of https://github.com/isso-comments/isso/discussions/1010

Please don't open multiple threads, it creates confusion when trying to find issues.
